test/src/loaders/ncbi_downloader/ncbi_downloader_test.py

```diff
@@ -43,16 +43,15 @@ def test_make_work_dir(setup_and_teardown):
     assert path.parents[0].name == source
 
 
-def test_fetch_gtdb_genome_ids(setup_and_teardown):
+def test_fetch_gtdb_genome_ids(setup_and_teardown, mocker):
     tmp_dir, script_file = setup_and_teardown
 
-    release_ver = '207'
```

Comment:
If we're going to only download from one release why 202 vs 207?

Reply:
hmmm I didn't get what you mean. I removed `207` hoping to speed up the test. Are you suggesting we should add it back?

Reply:
I was wondering why you picked to test 202 rather than testing 207.

That said, maybe we should just leave the tests as is. They only take 30s on GHA, and the last couple tests I've run locally have been <90s

Reply:
👍  revert changes to tests. It takes about 30s run locally.

Reply:
:+1: